README.md

```diff
@@ -134,3 +140,52 @@ haskell_library(
     </tr>
   </tbody>
 </table>
+
+### haskell_binary
+
+Imports a prebuilt shared library.
```

Comment:
Maybe add a sentence that begins, "Use this to …"? Suggesting this because I'm not 100% sure of when I need to use this.